app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterYourContactDetailsController.scala

```diff
@@ -323,33 +321,29 @@ object EnterClaimantDetailsAsImporterCompanyController {
 
   val claimantDetailsAsImporterCompanyForm: Form[ClaimantDetailsAsImporterCompany] = Form(
     mapping(
-      "enter-claimant-details-importer-company.importer-company-name" -> nonEmptyText(maxLength = 512),
-      "enter-claimant-details-importer-company.importer-email"        -> Email.mappingMaxLength,
-      "enter-claimant-details-importer-company.importer-phone-number" -> PhoneNumber.mapping,
-      ""                                                              -> Address.nonUkAddressFormMapping
+      "enter-your-contact-details.importer-company-name" -> nonEmptyText(maxLength = 512),
```

Comment:
we can remove the references to importer here

so just `enter-your-contact-details.company-name` etc